### PR TITLE
Bug 157608, Removed filename key and value

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1576088

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@cevich PTAL. Is this change for 3.10 only, or should it also be applied to 3.9? Thanks!

**Comment by cevich** [comment]
@ahardin-rh IDK :disappointed: I only tested 3.10, not sure when that option was removed.  @sdodson ?

**Comment by ahardin-rh** [comment]
@jianlinliu Please provide a QE review

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -1936,7 +1936,7 @@ openshift_deployment_type=origin
 endif::[]
 
 # uncomment the following to enable htpasswd authentication; defaults to DenyAllPasswordIdentityProvider
-#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', '/etc/origin/master/htpasswd'}]
```
Comment (cevich) [inline install_config/install/advanced_install.adoc]: This is a JSON key, so the value's gotta be clobbered as well:

```YAML
#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider'}]
```

Comment (cevich) [reply install_config/install/advanced_install.adoc]: same fore below.

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -1936,7 +1936,7 @@ openshift_deployment_type=origin
...
```
Comment (michaelgugino) [inline install_config/install/advanced_install.adoc]: Remove string '/etc/origin/master/htpasswd'

The path is hard-coded now.

**Review by michaelgugino — CHANGES_REQUESTED** [review]

File: install_config/install/advanced_install.adoc (hunk 2 above)
Comment (cevich) [reply install_config/install/advanced_install.adoc]: Ya, that's what I was trying to say also :smile:

**Comment by cevich** [comment]
@michaelgugino off-hand, do you know when that value was hard-coded?  Was it for 3.9 also?

**Comment by ahardin-rh** [comment]
@cevich @michaelgugino Thanks! I think I got them all now :beers:

**Comment by cevich** [comment]
LGTM (FWIW)

**Comment by ahardin-rh** [comment]
Looks like this is for 3.10+ https://bugzilla.redhat.com/show_bug.cgi?id=1576088#c4

**Comment by ahardin-rh** [comment]
QE approved in the BZ; no peer review needed

**Pull request merged**